vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -284,16 +307,15 @@ ent config --set {key} {value}
 ```
 ent config --set {key}
 ```
-
 ### Keys
 | Key  | Definition  |
 |---|---|
-| ENTANDO_NAMESPACE  |  stores the fallback namespace used by explicit or implicit runs of "ent kubectl" |
-|  ENTANDO_APPNAME | stores the EntandoApp name related to the current profile |
+| ENTANDO_NAMESPACE  |  stores the fallback namespace used by explicit or implicit runs of `ent kubectl` |
+|  ENTANDO_APPNAME | stores the Entando Application name related to the current profile |
 |  DESIGNATED_JAVA_HOME | stores the path of the Java version used internally by ent |
 
 ## Diagnostic Commands
-Performing the Automatic Install found in the Getting Started guide installs the CLI in a Multipass VM. The following commands can be run from this VM for insight into an Entando Application.
+Performing the [Automatic Install](../../docs/getting-started/README.md#automatic-install) found in the Getting Started guide installs the CLI in a Multipass VM. The following commands can be run from this VM for insight into an Entando Application.
```

Comment:
insight >> diagnostics ?
insight makes it mysterious

Reply:
diagnostics? no insight is correct ?? like doing it manually you learn about what's going on, giving you insight into the process ??